### PR TITLE
Simplify Contributing and PR templates

### PR DESCRIPTION
**Pull request by probablykasper**
A bunch of changes to the guidelines and README.

To avoid confusion (and people skipping stuff), `CONTRIBUTING.md` has the same content as the pull request template. It might be a little weird this way, but it makes the process way easier. There's still 14 different guidelines just to add an app, but it's better than the 30 you had to read previously.

Some guidelines were combined or removed when another guideline already pretty much covered it.

### CODE REVIEW

File: .github/contributing.md [hunk 1]
```diff
@@ -1,32 +1,44 @@
...
+- [ ] Add entries to the bottom of the correct category.
+- [ ] No unnecessary words like `for Tauri`, `a Tauri plugin` and `Super-Fast`.
+- [ ] Description does not start with `A` or `An`.
+- [ ] Description is under 24 words.
+- [ ] Description has no links or parenthesis.
+- [ ] Use `backticks` when mentioning package names.
+- [ ] Double-check spelling and grammar.
+- [ ] No trailing whitespace is added to the end of any file.
+- [ ] One suggestion per pull request.
+- [ ] I understand that entries can be removed if they do not follow the guidelines.
```
Comment (amrbashir) [inline .github/contributing.md]:
```suggestion
- Use the following format: `[Title](link) - Description.`
- Add entries to the bottom of the correct category.
- No unnecessary words like `for Tauri`, `a Tauri plugin` and `Super-Fast`.
- Description does not start with `A` or `An`.
- Description is under 24 words.
- Description has no links or parenthesis.
- Use `backticks` when mentioning package names.
- Double-check spelling and grammar.
- No trailing whitespace is added to the end of any file.
- One suggestion per pull request.
- I understand that entries can be removed if they do not follow the guidelines.
```

File: .github/contributing.md [hunk 2]
```diff
@@ -1,32 +1,44 @@
...
 
-Thank you for your suggestion!
+<!-- Ignore unless you're contributing to Plugins/Integrations -->
 
-## Apps
+- [ ] Works with **Tauri 1.x and onward**.
+- [ ] The project is open source and accepts contributions.
+- [ ] The repo is at least 30 days old.
+- [ ] Documentation is in English.
+- [ ] The project is active and maintained.
```
Comment (amrbashir) [inline .github/contributing.md]:
```suggestion
- Works with **Tauri 1.x and onward**.
- The project is open source and accepts contributions.
- The repo is at least 30 days old.
- Documentation is in English.
- The project is active and maintained.
```

File: .github/contributing.md [hunk 3]
```diff
@@ -1,32 +1,44 @@
...
 
-- The app is using Tauri.
-- The app is original and not too simple.
-- Tauri strives to enable fast, lightweight and secure apps. Therefore we expect featured apps to make a (reasonable) effort to be fast, lightweight and secure.
-- If you're submitting a closed source app, include evidence of it being built with Tauri.
+- [ ] Works with **Tauri 1.x and onward**.
+- [ ] The repo is at least 30 days old.
+- [ ] Documentation is in English.
+- [ ] The template provides enough information about how to get started and what's included.
+- [ ] The template is pretty different from the existing templates.
```
Comment (amrbashir) [inline .github/contributing.md]:
```suggestion
- Works with **Tauri 1.x and onward**.
- The repo is at least 30 days old.
- Documentation is in English.
- The template provides enough information about how to get started and what's included.
- The template is pretty different from the existing templates.
```

File: .github/contributing.md [hunk 4]
```diff
@@ -1,32 +1,44 @@
...
+## Apps
 
-## Violations
+<!-- Ignore unless you're contributing to Apps -->
 
-Please notice that entries that violate any of the above guidelines can be removed. We tend to be lenient and give you the benefit of the doubt, but spam, malware, projects not using tauri or repeat offenders will get removed.
+- [ ] The app is original and not too simple.
+- [ ] The README is in English.
+- [ ] The app makes a reasonable effort to be fast, lightweight and secure.
+- [ ] If the app closed source, evidence of it being built with Tauri is included.
```
Comment (amrbashir) [inline .github/contributing.md]:
```suggestion
- The app is original and not too simple.
- The README is in English.
- The app makes a reasonable effort to be fast, lightweight and secure.
- If the app is closed source, evidence of it being built with Tauri is included.
```

**Review by amrbashir — CHANGES_REQUESTED** [review]

**Review by amrbashir — APPROVED** [review]

**Comment by JonasKruckenberg** [comment]
Tbh there are things missing in the reworked guidelines and PR template that have immediately caused problems: https://github.com/tauri-apps/awesome-tauri/pull/46

But I will re-add them in a separate PR

**Pull request merged**